validator/tests/sawtooth_suites/ts_nightly_poet1.py

```diff
@@ -45,7 +49,8 @@ def test_suite(self):
             suite.addTest(TestConvergence('test_bootstrap', urls))
             suite.addTest(TestIntegration('test_intkey_load_ext', urls))
             suite.addTest(TestIntegration('test_missing_dependencies', urls))
-            suite.addTest(TestSawtoothStats('test_sawtooth_stats', urls))
+            suite.addTest(TestSawtoothStatsTwistedThread('test_sawtooth_stats_twisted_thread', urls))
+            suite.addTest(TestPoet1IntLoad('test_poet1_int_load', urls))
```

Comment:
Andrea, there were some rebase errors that wouldn't let me update this branch and include other commits from 0.7, so i have created a new branch here https://github.com/hyperledger/sawtooth-core/pull/140. I will be closing this one later today.